app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -277,10 +278,14 @@ class DefaultToolbarMenu(
         onItemTapped.invoke(ToolbarMenu.Item.Quit)
     }
 
-    private val readerMode = BrowserMenuImageSwitch(
+    private val readerMode = BrowserMenuHighlightableSwitch(
         label = context.getString(R.string.browser_menu_read),
-        imageResource = R.drawable.ic_readermode,
-        initialState = readerModeStateProvider
+        startImageResource = R.drawable.ic_readermode,
+        initialState = readerModeStateProvider,
+        highlight = BrowserMenuHighlight.LowPriority(
+            label = context.getString(R.string.browser_menu_read),
+            notificationTint = getColor(context, R.color.whats_new_notification_color)
+        )
```

Comment:
`isHighlighted` should also be set. I don't think the menu currently checks visibility.

Reply:
`isHighlighted` defaults to true, which is what we always want here.  But it's probably not a bad idea to put it in here to be explicit.

Reply:
If you include these options in the toolbar list when reader mode is unavailable, then the highlight may always appear.

Reply:
Oh, I see what you mean.  Thanks for clarifying.

Reply:
We looked into this and it seems that it does filter is visibility when adding the highlight to the three dot menu, so I'm resolving this comment.